script.js

```diff
@@ -0,0 +1,10 @@
+function sum() {
+    var a,b;
+        var a =Number(document.getElementById("a").value)
+        var b =Number(document.getElementById("b").value)
+     sum = a + b 
+        var answer = parseInt(a) + parseInt(b)
```

Comment:
You have two lines that do the same thing. You have one variable named "sum" and one named "answer". You aren't ever using sum. Also, don't declare a variable sum without using var or some other keyword (you'll learn those later).

You do not need to parse the numbers into integers. You have already called Number() on your html value strings to turn them into numbers. parseInt() just turns those numbers into integers, which you should do instead of Number() if that was what you wanted in the first place. Only do one.